src/subtitling/Group setups/github.md

```diff
@@ -0,0 +1,145 @@
+You'll want to give them either "write" or "maintain" roles
+so they'll be able to properly make changes without many headaches.
+
+![Managing access](../images/gh_manage_access.png)
+
+
+[GitHub_home]: (https://github.com/)
+[GitLab_home]: (https://about.gitlab.com/)
+[BitBucket_home]: (https://bitbucket.org/)
+
+## Basic actions
+
+**If you are new to GitHub,
+we highly recommend you use the [GitHub desktop client](GitHub_desktop)**
+(this also works with other,
+similar platforms).
+This guide was written under the assumption that most users will be using this client.
+
+### Cloning
+
+After installing the desktop client,
+open it and log in.
+You can then clone the repository to your system by going to `File > Clone repository`,
+or by going to the repository on GitHub and going `Code > Open with GitHub Desktop`.
+
+![Opening a repo with the desktop client](../images/gh_clone.png)
+
+If the repository has not been set up yet,
+you can set it up in your desktop client through the link on the repository.
+This will automatically open up the cloning menu in your desktop client.
+
+![Setting up a repo with the desktop client](../images/gh_setup_repo.png)
+
+### Syncing file changes
+
+When editing files from a Git instance,
+Git will remember and verify what files have been changed.
+Before making any commits however,
+you'll want to first double-check that you have every file pulled to your local repository.
+Click "Fetch origin" at the top,
+and pull in any newly-pushed commits made by other users.
+
+Now you can verify your own changes and commit the files you want to upload/update in the repository.
+The desktop client will additionally give you a very useful display of what is different between your file
+and the latest version of the file you pulled in from the repo.
+To make things easier for your teammates,
+you'll want to make sure to give your commit a helpful title and description if necessary.
+Finally,
+press "Commit to **master**",
+and once you've finished making all your changes and committed them,
+press "Push origin".
+
+![Pulling in changes made to the repository](../images/gh_syncing.png)
+
+
+If you want to check the history to see what your teammates have changed,
+you can view the history by clicking the "History" tab.
+This will give you a useful oversight of all the commits,
+as well as their titles/descriptions and what changes have been made to the files.
+
+![Viewing the repository's history](../images/gh_history.png)
+
+[GitHub_desktop]: (https://desktop.github.com/)
+
+## Resolving merge conflicts
+
+There are going to be times when multiple users accidentally write to the same file at the same time.
+This is usually no issue,
+but you may from time to time run into a merge conflict.
+This means that changes made by multiple users are incompatible with one another,
+and will require manual fixing.
+
+To resolve these,
+you'll want to open up the files with merge conflicts in your text editor
```

Comment:
Does Github desktop not come with a merge resolver? Regardless, the mention not to open in Aegisub is very important indeed.

Reply:
It doesn't, it asks you to open a text editor instead.

Reply:
Kinda disappointing, but workable.